backend/src/test/java/com/woowacourse/momo/service/meeting/MeetingServiceTest.java

```diff
@@ -87,4 +89,76 @@ void findByUUID() {
         softAssertions.assertThat(result.availableDates().get(0)).isEqualTo(LocalDate.now().minusDays(1));
         softAssertions.assertAll();
     }
+
+    @DisplayName("약속 정보와 참가자 정보를 통해 약속을 등록한다.")
+    @Test
+    void create() {
+        //given
+        MeetingCreateRequest request = new MeetingCreateRequest(
+                "momoHost",
+                "momo",
+                "momoMeeting",
+                8,
+                List.of(LocalDate.of(2024, 7, 24), LocalDate.of(2024, 7, 25)),
+                LocalTime.of(8, 0),
+                LocalTime.of(22, 0));
+
+        //when
+        meetingService.create(request);
+
+        //then
+        List<Meeting> meetings = meetingRepository.findAll();
+        List<Attendee> attendees = attendeeRepository.findAll();
+        List<AvailableDate> availableDates = availableDateRepository.findAll();
+
+        assertAll(
+                () -> assertThat(meetings).hasSize(1),
+                () -> assertThat(attendees).hasSize(1),
+                () -> assertThat(availableDates).hasSize(2)
+        );
+    }
+
+    @DisplayName("약속을 생성한 참가자의 역할은 호스트이다.")
+    @Test
+    void createdMeetingByHost() {
+        //given
+        MeetingCreateRequest request = new MeetingCreateRequest(
+                "momoHost",
+                "momo",
+                "momoMeeting",
+                8,
+                List.of(LocalDate.of(2024, 7, 24), LocalDate.of(2024, 7, 25)),
+                LocalTime.of(8, 0),
+                LocalTime.of(22, 0));
+
+        //when
+        meetingService.create(request);
+
+        //then
+        List<Meeting> meetings = meetingRepository.findAll();
+        Attendee attendee = attendeeRepository.findByMeetingAndName(meetings.get(0), new AttendeeName("momoHost"))
+                .orElseThrow();
+
+        assertThat(attendee).extracting("role").isEqualTo(Role.HOST);
+    }
+
+    @DisplayName("약속을 생성할 때 같은 약속일을 2번 이상 보내면 예외가 발생합니다.")
+    @Test
+    void throwExceptionWhenDuplicatedDates() {
+        //given
+        LocalDate date = LocalDate.of(2024, 7, 24);
+        MeetingCreateRequest request = new MeetingCreateRequest(
+                "momoHost",
+                "momo",
+                "momoMeeting",
+                8,
+                List.of(date, date),
+                LocalTime.of(8, 0),
+                LocalTime.of(22, 0));
+
+        //when
+
+        //then
+        assertThatThrownBy(() -> meetingService.create(request)).isInstanceOf(MomoException.class);
+    }
```

Comment:
메서드 체인으로 보여 아래처럼 줄바꿈이 필요해 보입니다!
```suggestion
        assertThatThrownBy(() -> meetingService.create(request))
                .isInstanceOf(MomoException.class);
    }
```